src/smart-components/request/action-modal.js

```diff
@@ -22,12 +22,12 @@ const ActionModal = ({
   actionType,
   createRequestAction,
   closeUrl,
-  postMethod
+  postMethod,
+  postMethodParameter
```

Comment:
This is not necessary we can just modify the post method since we are not generating/altering the `postMethodParameter` in any way in this component. Just pass the `postMethod` like this:

```jsx
<Foo postMethod={() => bar('params')} />
```